tests/thrift/thrift_test.py

```diff
@@ -24,6 +25,48 @@ def test_create_span():
     assert span.binary_annotations == 'binary_ann'
     assert span.timestamp == 1485920381.2 * 1000000
     assert span.duration == 2.0 * 1000000
+    assert span.trace_id_high is None
+
+
+def test_create_span_with_128_bit_trace_ids():
+    span = thrift.create_span(
+        span_id='0000000000000001',
+        parent_span_id='0000000000000002',
+        trace_id='000000000000000f000000000000000f',
+        span_name='foo',
+        annotations='ann',
+        binary_annotations='binary_ann',
+        timestamp_s=1485920381.2,
+        duration_s=2.0,
+    )
+    assert span.trace_id == 15
+    assert span.trace_id_high == 15
```

Comment:
Can you have the the high order portion and low order portion be different so we can be sure that the extraction is correct?